docs/proposals-done/202106-proposals-process.md

```diff
@@ -0,0 +1,49 @@
+---
+type: proposal
+title: Proposal Process
+status: complete
+owner: bwplotka
+menu: proposals-done
+---
+
+# 2021-06: Proposal Process
+
+* **Owners:**:
+  * [`@bwplotka`](https://github.com/bwplotka)
+
+* **Other docs:**
+  * [KEP Process](https://github.com/kubernetes/enhancements/blob/master/keps/README.md)
+  * [RHOBS Handbook Proposal Process](https://github.com/rhobs/handbook/pull/15)
+
+> TL;DR: We would like to propose an improved, official proposal process for Monitoring Group that clearly states when, where and how to create proposal/enhancement/design documents.
+
+## Why
+
+More extensive architectural, process, or feature decisions are hard to explain, understand and discuss. It takes a lot of time to describe the idea, to motivate interested parties to review it, give feedback and approve. That's why it is essential to streamline the proposal process.
```

Comment:
Maybe we can vaguely describe _when_ such a design doc is needed? I think describing this should be pretty much necessary given that we want to avoid bureaucracy as much as possible, right?

Reply:
I presented that in the flow, but I can repeat here too 🤗

Reply:
Added section to proposal process